package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "angular-routing-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot"
```

Comment:
"lint": "./node_modules/eslint/bin/eslint.js ./**/*.js"     forgot to add your lint script